crates/host/src/wasmbus/mod.rs

```diff
@@ -2868,7 +2868,10 @@ impl Host {
                 // TODO(#1548): Providers should not receive [wasmcloud_control_interface::InterfaceLinkDefinition]s
                 // they should receive a list of named configurations to use
                 link_definitions: vec![],
-                config: config.get_config().await.clone(),
+                configs: HashMap::from([(
+                    "default".to_string(),
+                    config.get_config().await.clone(),
+                )]),
```

Comment:
This should be different, the configuration as passed to the provider isn't based on a link name, it's the set of component configuration for that entire provider instance. The way I see it is that this parameter, `config`, should be that piece of static config (for example, a default connection address for every link), but we should have another parameter called `link_configs` which entirely replaces `link_definitions`. Does that make sense?

Reply:
Sure -- I'm a bit confused on how the static config works for multiple link names that come through, but I get that it's not supposed to be stored here -- I've reverted this